app/mailers/candidate_mailer.rb

```diff
@@ -295,6 +295,30 @@ def offer_accepted(application_choice)
     )
   end
 
+  def ucas_match_initial_email_duplicate_applications(application_choice)
```

Comment:
Would it make more sense for this method to take the `UCASMatch` itself? Then no need to depend on `TimeLimitCalculator` out here too — could just ask the match to figure it out

Reply:
I did initially pass in UCASMatch but then decided against it so it would follow the same pattern as the rests of the CandidateMailer methods. I would just argue that this seems to be the purpose of the TimeLimitCalculator anyway, and it is also reused in the CandidateMailer so it makes sense to keep things consistent.

We also don't use any other information found on the UCASMatch itself.